src/DynamoPublish/Properties/Resources.resx

```diff
@@ -219,4 +219,16 @@
   <data name="ManagerErrorMessage" xml:space="preserve">
     <value>"Manager" is not set. Please, set manager in configuration file.</value>
   </data>
+  <data name="RequestApproved" xml:space="preserve">
+    <value>Your request is already approved.</value>
+  </data>
+  <data name="RequestOnPendingState" xml:space="preserve">
+    <value>Request pending...</value>
+  </data>
+  <data name="ApprovedStatus" xml:space="preserve">
+    <value>approved</value>
+  </data>
+  <data name="PendingStatus" xml:space="preserve">
+    <value>pending</value>
+  </data>
```

Comment:
All strings in this file are candidates for localization. These two status values come down from the server right? I don't think they should be localized.